llvm/include/llvm/DebugInfo/LogicalView/Core/LVScope.h

```diff
@@ -826,6 +834,23 @@ class LVScopeTemplatePack final : public LVScope {
   void printExtra(raw_ostream &OS, bool Full = true) const override;
 };
 
+// Class to represent a DWARF Module.
+class LVScopeModule final : public LVScope {
+public:
+  LVScopeModule() : LVScope() {
+    setIsModule();
+    setIsLexicalBlock();
+  }
+  LVScopeModule(const LVScopeModule &) = delete;
+  LVScopeModule &operator=(const LVScopeModule &) = delete;
+  ~LVScopeModule() = default;
+
+  // Returns true if current scope is logically equal to the given 'Scope'.
+  bool equals(const LVScope *Scope) const override;
+
+  void printExtra(raw_ostream &OS, bool Full = true) const override;
+};
```

Comment:
From the included test case:
```
DW_TAG_module
      DW_AT_name	("DebugModule")
      DW_AT_LLVM_config_macros	("-DMODULES=0")
      DW_AT_LLVM_include_path	("/llvm/tools/clang/test/Modules/Inputs")
      DW_AT_LLVM_apinotes	("m.apinotes")
```
Any additional information that can be collected for the `LVScopeModule`?

Reply:
I believe not (besides the attributes that are already parsed in https://github.com/llvm/llvm-project/blob/main/llvm/lib/DebugInfo/LogicalView/Readers/LVDWARFReader.cpp#L245).  IMHO, not very interesting ATM to collect LLVM-specific DWARF attributes.